content/en/docs/refguide/modeling/application-logic/workflows/perform-workflow-basic-functions.md

```diff
@@ -101,6 +102,9 @@ When using cut/copy/paste, note the following:
 * When the clipboard contents has an **Annotation**, it can only be pasted into the workflow or into individual activities that support annotations.
 * Pasting an activity after an **End** or **Jump** activity results in a consistency error as an **End** or **Jump** activity should be placed at the end of a flow.
 * Pasting activity can result in a consistency error when the next activity becomes unreachable by either placing an **End** or **Jump** activity in front of it or when all flows before a merge end with an **End** or **Jump** activity.
+* Pasting a boundary event on a **User Task**, **Wait For Notification**, **Call Microflow**, **Call Workflow** adds it to the activity, provided there are less than five.
+* Pasting a **Jump** directly inside a non-interrupting boundary event flow will result in a consistency error. However, pasting it is allowed on outgoing flows from activities except parallel split paths.
```

Comment:
Can we combine these lines and explain when it's possible (the initial sentence might scare readers off)? Pasting a Jump inside a non-interrupting boundary path is (only) possible when there are multiple flows going to the boundary path end (e.g. a user task with multiple outcomes). 
A screenshot of what is not possible and what is possible would help

Reply:
I added another description of why this is not allowed, and made it a bit more clear.